pytensor/tensor/slinalg.py

```diff
@@ -1669,6 +1670,73 @@ def block_diag(*matrices: TensorVariable):
     return _block_diagonal_matrix(*matrices)
 
 
+class BandedDot(Op):
```

Comment:
Put in blas.py?

Reply:
I saw your message, fine

Reply:
You mean in `pytensor.tensor.blas` ? I can do that if you think it's better